web/tests/functional/report_viewer_api/test_report_counting.py

```diff
@@ -76,9 +76,10 @@ def setUp(self):
              'deadcode.DeadStores': 6,
              'unix.MismatchedDeallocator': 1}
 
-        self.run1_sev_counts = {Severity.MEDIUM: 1,
+        self.run1_sev_counts = {Severity.UNSPECIFIED: 3,
+                                Severity.MEDIUM: 1,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Severity' may be undefined, or defined from star imports: codeCheckerDBAccess_v6.ttypes](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4575450)